pkg/daemon/plugin.go

```diff
@@ -1,11 +1,12 @@
 package daemon
 
 import (
-	"fmt"
-	"plugin"
-
-	"github.com/golang/glog"
 	sriovnetworkv1 "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
+	genericplugin "github.com/k8snetworkplumbingwg/sriov-network-operator/pkg/plugins/generic"
+	intelplugin "github.com/k8snetworkplumbingwg/sriov-network-operator/pkg/plugins/intel"
+	k8splugin "github.com/k8snetworkplumbingwg/sriov-network-operator/pkg/plugins/k8s"
```

Comment:
I have a little concern about this import, as the `k8s.init()` func contains a [panic call](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/ba5ad00dccdb2c5949a507af3ce9d1787c79e4ee/pkg/plugins/k8s/k8s_plugin.go#L104). I'm not sure about how likely it is going to happen in production, but prior to this changes the init func was called only 

```
if utils.ClusterType != utils.ClusterTypeOpenshift {
```

Reply:
That's good. thanks